docker/scripts/ld.command.init.sh

```diff
@@ -11,9 +11,22 @@ function ld_command_init_exec() {
         echo -e "${BRed}This project is already initialized. ${Color_Off}"
         echo -e "${Yellow}Are you really sure you want to re-initialize the project? ${Color_Off}"
         read -p "[yes/NO] " ANSWER
+        # Lowercase.
+        ANSWER="$(echo ${ANSWER} | tr [A-Z] [a-z])"
         case "$ANSWER" in
             'y'|'yes')
-                echo -e "${Green}Sure, thanks for the confirmation.${Color_Off}"
+                if [ -e "$DOCKERSYNC_FILE" ] || [ -e "$DOCKER_COMPOSE_FILE" ]; then
+                    echo -e "${BYellow}WARNING: ${Yellow}There is docker-compose and/or docker-sync configuration (.yml) files in project root.${Color_Off}"
+                    echo -e "${Yellow}If you continue and rename your volumes ${BYellow}you may loose data (database).${Yellow}.${Color_Off}"
```

Comment:
```suggestion
                    echo -e "${Yellow}If you continue and rename your volumes ${BYellow}you may lose data (database).${Color_Off}"
```
One typo, one c-p error?